### ISSUE TITLE
cart qty not updating

### ISSUE THREAD

**Issue by yamineesowri** [husseycoding/ajaxcartpopup#2]
I am using ajax cart popup extension.
Everything is working fine.
But the cart quantity is not updating after clicking add to cart button.
when reloading page the only it's updating.
I want the fix to update the qty through ajax.

**Comment by husseycoding** [comment]
Do you see the same problem on a clean install with default theme?  I am assuming you are talking about the header link?  If so my guess would be that your theme changes the HTML here away from the default so that the extension is no longer able to update it.  You should restore the default HTML structure here and then the quantity updating should work correctly.

**Comment by yamineesowri** [comment]
yes it's working with the rwd theme but i am using ultimo theme.
http://escapade.store.emperium.net/
can you suggest me how to fix in this theme.

**Comment by husseycoding** [comment]
The standard RWD theme HTML for the header cart link is:

    <a class="skip-link skip-cart  no-count" href="#header-cart">
        <span class="icon"></span>
        <span class="label">Cart</span>
        <span class="count">0</span>
    </a>

You need to match this in your theme to have it work correctly

**Comment by husseycoding** [comment]
Probably you can just add the class `top-link-cart` to the element which currently has an id of `cart`.

**Comment by yamineesowri** [comment]
i have added the class but not working
see this is my cart code

<div id="mini-cart-wrapper-regular">
<div id="mini-cart" class="mini-cart dropdown<?php echo $cartClasses; ?>">

    <a href="#header-cart" class="mini-cart-heading dropdown-heading cover skip-link skip-cart top-link-cart">
        <span>
        
            <span class="icon ic ic-cart"></span>
            <?php if ($_cartQty > 0): ?>
                <span class="count"><?php echo $_cartQty; ?></span>
            <?php endif; ?>
            <span class="label<?php if (!$showLabel) echo ' no-display'; ?>"><?php echo $this->__('Cart'); ?></span>
            <?php /* Comment-out to remove the subtotal from the mini-cart heading
            <?php if ($_cartQty > 0): ?>
                <?php if (!$canApplyMsrp): ?>
                    <span class="summary hide-below-960"><?php echo Mage::helper('checkout')->formatPrice($this->getSubtotal()); ?></span>
                <?php endif; ?>
            <?php else: ?>
                <span class="summary hide-below-960"><?php echo Mage::helper('checkout')->formatPrice(0); ?></span>
            <?php endif; ?>
            */ ?>
            <span class="caret">&nbsp;</span>

        </span>
    </a>

**Comment by yamineesowri** [comment]
if you confident to solve this issue.

I will give you the ftp access also.

Please respond as soon as possible.

**Comment by husseycoding** [comment]
Sorry my reply above was actually meant for another ticket.  If you want to have us look into your problem we can do but it won't be free.   If you are interested in paid support  contact us at support@husseycoding.co.uk.
